docs/modules/ROOT/partials/install/finalize_part1.adoc

```diff
@@ -72,27 +72,40 @@ done
 ----
 endif::[]
 
-=== Store the cluster's admin credentials in the password manager
+=== Ensure emergency admin access to the cluster
 
-. Once the cluster's production API certificate has been deployed, edit the cluster's admin kubeconfig file to remove the initial API certificate CA.
+. Check that emergency credentials were uploaded and are accessible:
 +
-[TIP]
+[source,bash]
+----
+emergency-credentials-receive "${CLUSTER_ID}"
+# Follow the instructions to use the downloaded kubeconfig file
+----
++
+[NOTE]
 ====
-You may see the error `Unable to connect to the server: x509: certificate signed by unknown authority` when executing `kubectl` or `oc` commands after the cluster's production API certificate has been deployed by Project Syn.
+You need to be in the passbolt group `VSHN On-Call`.
+
+If the command fails, check if the controller is already deployed, running, and if the credentials are uploaded:
 
-This error can be addressed by removing the initial CA certificate data from the admin kubeconfig as shown in this step.
+[source,bash]
+----
+kubectl --as=system:admin -n appuio-emergency-credentials-controller get emergencyaccounts.cluster.appuio.io -o=jsonpath='{range .items[*]}{.metadata.name}{"\t"}{.status.lastTokenCreationTimestamp}{"\n"}{end}'
```

Comment:
is there as specific reason to use `--as=system:admin` here?

Reply:
nope copy-paste oversight